terraform/deployments/ga4-analytics/main.tf

```diff
@@ -0,0 +1,26 @@
+terraform {
+  cloud {
+    organization = "govuk"
+    workspaces {
+      tags = ["gcp-ga4-analytics", "gcp", "production"]
+    }
+  }
+  required_version = "~> 1.5"
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = "~> 5.26.0"
+    }
+  }
+}
+
+provider "google" {
+  project = "ga4-analytics-352613"
+}
+
+resource "google_project" "project" {
+  name            = "GA4 Analytics"
+  project_id      = "ga4-analytics-352613"
+  folder_id       = "278098142879"
+  billing_account = "015C7A-FAF970-B0D375"
+}
```

Comment:
Has the project already been made - do we want the project itself managed by terraform? Maybe use a [data block instead](https://registry.terraform.io/providers/hashicorp/google/latest/docs/data-sources/project) or even just have a variable?